kolena/_api/v1/core.py

```diff
@@ -29,7 +30,7 @@ class CreateRequest:
         name: str
         metadata: Dict[str, Any]
         workflow: str
-        tags: Optional[List[str]] = None
+        tags: Optional[Union[List[str], Set[str]]] = None
```

Comment:
Why is this change needed?

Reply:
Removed and addressed need for this change elsewhere.